storage/rocksdb/ha_rocksdb.cc

```diff
@@ -704,9 +735,9 @@ static int rocksdb_trace_block_cache_access(
         "RocksDB: Failed to parse trace option string: %s. The correct "
         "format is sampling_frequency:max_trace_file_size:trace_file_name. "
         "sampling_frequency and max_trace_file_size are positive integers. "
-        "The block accesses are saved to the "
-        "rocksdb_datadir/block_cache_traces/trace_file_name.\n",
-        trace_opt_str.c_str());
+        "The block accesses or quries are saved to the "
```

Comment:
s/quries/queries

Reply:
Thank you @georgelorch, it comes from upstream, as it is a bit of work to fix it (we have resolve some conflicts merge percona/5.7), I will fix it in a future commit.